Util-JsonApiSerializer/Serialization/JsonApiTransformer.cs

```diff
@@ -24,12 +27,14 @@ public JsonSerializer Serializer
         public CompoundDocument Transform(object objectGraph, Context context)
         {
             Type innerObjectType = TransformationHelper.GetObjectType(objectGraph);
-
+            // note there appears to be no structure of HttpError in .NET Core
+#if NETCOREAPP
+#else
```

Comment:
@PaulRose112358 I believe you could do `#if !NETCOREAPP` here

Reply:
I was trying to be consistent throughout all of the code with the same if/then/else